tabular/src/autogluon/tabular/models/text_prediction/text_prediction_v1_model.py

```diff
@@ -17,7 +17,7 @@
 logger = logging.getLogger(__name__)
 
 
-class TextPredictionV1Model(AbstractModel):
+class AGTextPredictionModel(AbstractModel):
```

Comment:
@jwmueller What do you think the name should be?

`AGTextPredictionModel`
`AGTextPredictorModel`
`TextPredictorModel`

Reply:
`TextPredictorModel` sounds best to me. Make sure to change it in all the tutorials/docs as well if you're going to change the name

Reply:
@sxjscience Lets change it to `TextPredictorModel`. That way we are more aligned in future if we add `TabularPredictorModel` and `ImagePredictorModel`.

Reply:
Updated.